src/main/java/CinemaRoomRestService/RestService/api/mapper/SeatMapper.java

```diff
@@ -0,0 +1,48 @@
+package CinemaRoomRestService.RestService.api.mapper;
+
+import CinemaRoomRestService.RestService.api.dto.SeatDTO;
+import CinemaRoomRestService.RestService.api.dto.StatisticsDTO;
+import CinemaRoomRestService.RestService.domain.entity.CinemaRoom;
+import CinemaRoomRestService.RestService.domain.entity.Seat;
+import CinemaRoomRestService.RestService.domain.entity.Statistics;
+import CinemaRoomRestService.RestService.domain.entity.TokenOfSeat;
+import CinemaRoomRestService.RestService.api.dto.CinemaRoomDTO;
+import CinemaRoomRestService.RestService.api.dto.TokenOfSeatDTO;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+public class SeatMapper {
+
+    public CinemaRoomDTO toCinemaRoomDTO(CinemaRoom cinemaRoom) {
```

Comment:
You have several domain objects, so you should write several Mappers, do not write everything in one mapper

Reply:
You also can inject one mapper into another

Reply:
See an exhample

public final class ControlsMapper {

    private OrganizationUnitMapper organizationUnitMapper;

    private static final ControlsDTO[] EMPTY_ARRAY = new ControlsDTO[0];

    public ControlsDTO[] toControlsDTO(Collection<Controls> controls) {
        return Objects.isNull(controls)
                ? EMPTY_ARRAY
                : controls.stream()
                    .map(this::toOneControlsDTO)
                    .toArray(ControlsDTO[]::new);
    }

    private ControlsDTO toOneControlsDTO(Controls controls) {
        return ControlsDTO.builder()
                .id(String.valueOf(controls.getId()))
                .name(controls.getName())
                .description(controls.getDescription())
                .owner(organizationUnitMapper.toOrganizationUnitDTO(controls.getOwner()))
                .build();
    }
}

Reply:
Also rename folder mapper -> mappers